central/convert/typetostorage/discovered_cluster.go

```diff
@@ -12,16 +12,21 @@ func DiscoveredCluster(cluster *discoveredclusters.DiscoveredCluster) *storage.D
 	storageConfig := &storage.DiscoveredCluster{
 		Id: discoveredclusters.GenerateDiscoveredClusterID(cluster),
 		Metadata: &storage.DiscoveredCluster_Metadata{
-			Id:                cluster.GetID(),
-			Name:              cluster.GetName(),
-			Type:              cluster.GetType(),
-			ProviderType:      cluster.GetProviderType(),
-			Region:            cluster.GetRegion(),
-			FirstDiscoveredAt: cluster.GetFirstDiscoveredAt(),
```

Comment:
Why this is removed?

Reply:
The type in the input parameter changed from `*types.Timestamp` to `*time.Time`.
There is a code block later in the function to set this field.